app/controllers/health_controller.rb

```diff
@@ -0,0 +1,15 @@
+class HealthController < ApplicationController
+  def workers
+    statuses = worker_health_checker.statuses
+
+    status = statuses.all?(&:healthy?) ? :ok : :internal_error
```

Comment:
Sure, you mean the `.all?` part and not the `:ok` vs `:internal_error` HTTP status right?